h2olog.cc

```diff
@@ -57,7 +57,12 @@ static void show_event_per_sec(h2o_tracer_t *tracer, time_t *t0)
             const char *iso8601format = "%FT%TZ";
             strftime(s, sizeof(s), iso8601format, &t);
 
-            fprintf(stderr, "%s %20lu events/s\n", s, c);
+            if (tracer->lost_count > 0) {
+                fprintf(stderr, "%s %20lu events/s (possibly lost %" PRIu64 " events)\n", s, c, tracer->lost_count);
```

Comment:
`c` type is `uint64_t` so use `%PRIu64` instead of `%lu`. I suppose `tracer->lost_count` is not printed correctly on 64bit platform by wrong stack handling in `fprintf`.

https://github.com/toru/h2olog/blob/78dbb567c8167d308dbb9ad1ba26d95b4d189c5b/h2olog.cc#L64

ditto

Reply:
Right! 37b8bc7